src/github/workflows/cabal.yml

```diff
@@ -59,6 +60,16 @@ jobs:
         ghc-version: ${{ matrix.ghc-ver }}
         cabal-version: ${{ matrix.cabal-ver }}
 
+    - name: Environment settings based on the Haskell setup
+      run: |
+        export GHC_VER=$(ghc --numeric-version)
+        echo "GHC_VER    = ${GHC_VER}"                 >> ${GITHUB_ENV}
```

Comment:
Need to remove the spaces around the '=' as the workflow runner does seem to take them seriously.

Reply:
Fixed.